packages/kolibri-tools/test/fixtures/TestUserPermissions.js

```diff
@@ -67,7 +68,7 @@ export function showUserPermissionsPage(store, userId) {
   const stopLoading = () => store.commit('CORE_SET_PAGE_LOADING', false);
 
   // Don't request any data if not an Admin
-  if (!store.getters.isSuperuser) {
+  if (!useUser().isSuperuser) {
```

Comment:
Again this would need to access `.value` as outside of the context of a component.